cdi/src/main/java/org/hibernate/validator/cdi/interceptor/spi/ValidationInterceptor.java

```diff
@@ -4,14 +4,16 @@
  * License: Apache License, Version 2.0
  * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
  */
-package org.hibernate.validator.cdi.internal.interceptor;
+package org.hibernate.validator.cdi.interceptor.spi;
```

Comment:
Any reason this cannot remain in an internal package *and* have access to reflection?

Reply:
well ... I mean ... 🙈 my thinking was that we don't really want to open or export something with `.internal` .. that's  until I've seen the problem with `ValidatorImpl` that it has to be opened 😖. Couldn't apply such logic there 😄